public/src/string-class.js

```diff
@@ -0,0 +1,196 @@
+const StringClass = {
+  /**
+   * has vowels
+   *
+   * Tests if a string contains vowels
+   *
+   * @return {Boolean} True or false depending on the match
+   */
+  hasVowels() {
+    return /[aeiou]/i.test(this);
+  },
+
+  /**
+   * to Upper
+   *
+   * converts strings to uppercase
+   *
+   * @return {String} string that has been converted to uppercase
+   */
+  toUpper() {
+    return this.replace(/[a-z]/g, (character) => {
+      return String.fromCharCode(character.charCodeAt() - 32);
+    });
+  },
+
+  /**
+   * to Lower
+   *
+   * converts strings to lowercase
+   *
+   * @return {String} string that has been converted to lowercase
+   */
+  toLower() {
+    return this.replace(/[A-Z]/g, (character) => {
+      return String.fromCharCode(character.charCodeAt() + 32);
+    });
+  },
+
+  /**
+   * upper case first
+   *
+   * converts strings to title case
+   *
+   * @return {String} string that has been converted to title case
+   */
+  ucFirst() {
+    return `${this[0].toUpper()}${this.slice(1)}`;
+  },
+
+  /**
+   * is question
+   *
+   * checks if a string is a question by looking for '?' sign
+   *
+   * @return {Boolean} true or false depending on the match
+   */
+  isQuestion() {
+    return /^[a-z0-9\s]+\?$/i.test(this.trim());
+  },
+
+ /**
+  * word
+  *
+  * checks for words in a string
+  *
+  * @return {String} returns array of words in the string
+  */
+  word() {
+    return this.replace(/[^\w\s]/g, '').split(/\s+/);
+  },
+
+  /**
+   * word count
+   *
+   * checks the number of words present in a string
+   *
+   * @return {Number} the number of words in the string
+   */
+  wordCount() {
+    return this.word().length;
+  },
+
+  /**
+   * to currency
+   *
+   * converts currency representation to string
+   *
+   * @return {String} currency representation of the the number in String
+   */
+  toCurrency() {
+    return Number(this)
+           ? Number(this).toFixed(2).replace(/(\d)(?=(\d{3})+\.)/g, '$1,')
+           : 'Invalid Input';
+  },
+
+  /**
+   * from currency
+   *
+   * converts strings representation of currency
+   *
+   * @return {Number} number representation of the currency String
+   */
+  fromCurrency() {
+    const result = Number(this.replace(/(,|\.00)/g, ''));
+    if (result) {
+      return result;
+    }
+    return 'Invalid Input';
```

Comment:
### Critical

Line 105 to 108 can be reduced to one line with the ternary operator.